src/drivers/Qt/HelpPages.cpp

```diff
@@ -62,7 +62,7 @@ void consoleWin_t::OpenHelpWindow(std::string subpage)
 
 	if ( helpFileName.length() == 0 )
 	{
-		#ifdef WIN32
+		#if defined(WIN32) && (QT_VERSION_MAJOR < 6)
```

Comment:
This would cause the Qt6 windows builds to look in the wrong place for the help.chm file. What does this have to do with Qt6?

Reply:
My mistake, I've reverted this.